File: ts/src/fizz_buzz.ts
function say(number: number) {
  let strReturn

  if (number % 15 == 0) {
    strReturn = "FizzBuzz"
  } else {
    if (number % 3 == 0) strReturn = "Fizz"

    if (number % 5 == 0) strReturn = "Buzz"
  }

  if (strReturn != undefined) return strReturn

  return number.toString()
}

export default say

File: java/src/main/java/assignment/FizzBuzz.java
package assignment;

public class FizzBuzz {
    public static String say(int number) {
        if (number % 15 == 0)
            return "FizzBuzz";

        if (number % 3 == 0)
            return "Fizz";

        if (number % 5 == 0)
            return "Buzz";

        return String.valueOf(number);
    }
}
